Error with @rlay/ontology: Cannot find module './rlay_ontology_js_nodejs_bg'

According to @MichaelHirn, the currently published version of `@rlay/ontology` throws an error on import:

```
Error: Cannot find module './rlay_ontology_js_nodejs_bg'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/mj/rlay-utils/rlay-client-lib/node_modules/@rlay/ontology/rlay_ontology_js_nodejs.js:1185:8)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
```

--- a/rlay_ontology_js/Cargo.toml
+++ b/rlay_ontology_js/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "rlay_ontology_js"
-version = "0.1.0-alpha.0"
+version = "0.1.0-alpha.1"
 authors = ["Maximilian Goisser <goisser94@gmail.com>"]
 edition = "2018"
 

--- a/rlay_ontology_js/prepare-pkg.js
+++ b/rlay_ontology_js/prepare-pkg.js
@@ -6,5 +6,6 @@ const packageJsonPath = path.join(__dirname, './pkg/package.json');
 const contents = JSON.parse(fs.readFileSync(packageJsonPath));
 // set name
 contents.name = '@rlay/ontology';
+contents.files.push("rlay_ontology_js_nodejs_bg.js");
 
 fs.writeFileSync(packageJsonPath, JSON.stringify(contents, null, 4));

